### ISSUE TITLE
invalid_doc_attributes does not run if there are no items in a crate

### ISSUE THREAD

**Issue by jyn514** [rust-lang/rust#85500]
This is probably not worth fixing.

I tried this code:

```rust
#![feature(no_core)]
#![no_core]
#![doc(x)]
```

I expected to see this happen: A warning that `doc(x)` is invalid.

Instead, this happened: No warning.

### Meta

`rustc --version --verbose`:  1.54.0-nightly (2021-05-18 4e3e6db011c5b482d2be

**Comment by jyn514** [comment]
Uhh it turns out this is actually invalid code, it only compiles when used with rustdoc because rustdoc has lots of hackery. Going to close this


```
error: requires `sized` lang_item 
```
